source/docs/03-metadata.xml

```diff
@@ -1392,9 +1392,19 @@
                         <specDesc key="colophon"/>
                      </specList>
                      </p>
-                  <p>The <gi scheme="MEI">condition</gi> element may contain text, or child elements that may be used for structuring the description (e.g., for linking) to accurately describe the physical state or condition of a source.</p>
-                     <p>The condition can thus be described in general (<gi scheme="MEI">physDesc</gi>) or in specific contexts (see “contained by” section of the element definition).</p>
-                     <p>For describing the condition of parts of a source (e.g., the binding) the <gi scheme="MEI">condition</gi> element can be used on the specific level. The level of detail you provide will depend on your specific needs for accurately recording the material condition.</p>
+                  <p>Encoding general information about the condition of a source.</p>
+                   <p>
+                      <specList>
+                        <specDesc key="condition"/>
+                      </specList>
+                   </p>
+                     <p>Every physical part of a source is in a certain condition. The element <gi scheme="MEI">condition</gi> may contain plain text but also provides several elements that can be used for structuring the text if necessary (e.g. for linking) in order to describe the state or condition of a source.</p>
+                     <p>The condition can thus be described in general (<gi scheme="MEI">physDesc</gi>) or in specific contexts (see element definition “contained by”).</p>
+                     <p>The <gi scheme="MEI">condition</gi> element as a direct child of <gi scheme="MEI">physDesc</gi> should be used for describing the condition of a source in general. For describing special concerns like the condition of special parts of a source (e.g. the binding) the condition element can be used on the specific level. This approach promotes machine readability of the encoded metadata. In the end it depends on your encoding guidelines how detailed the description of the material condition is organized.</p>
```

Comment:
```suggestion
                     <p>The <gi scheme="MEI">condition</gi> element as a direct child of <gi scheme="MEI">physDesc</gi> should be used to describe the condition of a source. To describe special concerns, such as the condition of parts of a source (e.g. the binding) the <gi scheme="MEI">condition</gi> element can be used on the specific level. This approach promotes machine readability of the encoded metadata. The detail you provide, and how the descriptions are structured, depend on your own encoding guidelines.</p>
```

Also "... used on the specific level" -- what level? Where?

"This approach promotes machine..." -- What approach?

Reply:
> Also "... used on the specific level" -- what level? Where?

See suggestion

> "This approach promotes machine..." -- What approach? 

I'll delete that sentence. Doesn't help here, as I see it now.

Including your suggestions:

```suggestion
                     <p>The <gi scheme="MEI">condition</gi> element as a direct child of <gi scheme="MEI">physDesc</gi> should be used to describe the condition of a source. To describe special concerns, such as the condition of parts or aspects of a source (e.g. the binding) the <gi scheme="MEI">condition</gi> element can also be used for a more detailed description of that part or aspect (e.g. at <gi scheme="MEI">binding</gi> ). The detail you provide, and how the descriptions are structured, depend on your own encoding guidelines.</p>
```